packages/cli/commands/theme/preview.js

```diff
@@ -32,21 +37,32 @@ const validateSrcPath = src => {
   return true;
 };
 
-exports.command = 'preview <src> <dest>';
-exports.describe = i18n(`${i18nKey}.describe`);
+exports.command = 'preview';
```

Comment:
Not a blocker, but you could update this to `preview [--src] [--dest]` to help make the options more discoverable.